app/models/manageiq/providers/azure/cloud_manager/security_group.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::Azure::CloudManager::SecurityGroup < ::SecurityGroup
+end
```

Comment:
@djberg96 -Do we need to create a subclass of SecurityGroup.
@blomquisg  - I have seen this before where we create an empty subclass, is there any value in that?